### PR TITLE
Add command to delete data from database

### PR DESCRIPTION
**Pull request by mveyrenc**
The command ''sidus:data:clean-data" can remove :
- data based en filter
- orphan data (data without attributes)
- data without family
- value without attribute

### CODE REVIEW

File: Command/CleanDataCommand.php [hunk 1]
```diff
@@ -0,0 +1,483 @@
+<?php
+/*
+ *    CleverAge/ProcessBundle
```
Comment (VincentChalnot) [inline Command/CleanDataCommand.php]: License does not correspond to this bundle ;)

File: Command/CleanDataCommand.php [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+                $queryBuilderAlias = uniqid('subQuery', false);
+                $familyCodeParam = 'familyCode'.$queryBuilderAlias;
+                $attributeCodeParam = 'attributeCode'.$queryBuilderAlias;
+
+                $queryBuilder = $this
+                    ->entityManager
+                    ->getRepository($this->valueClass)
+                    ->createQueryBuilder($queryBuilderAlias)
+                    ->select(
+                        sprintf('IDENTITY(%s.%s)', $queryBuilderAlias, $targetAttribute->getType()->getDatabaseType())
```
Comment (VincentChalnot) [inline Command/CleanDataCommand.php:275]: OMG

File: Command/CleanDataCommand.php (hunk 1 above)
Comment (mveyrenc) [reply Command/CleanDataCommand.php]: Fixed

File: Command/CleanDataCommand.php (hunk 2 above)
Comment (mveyrenc) [reply Command/CleanDataCommand.php:275]: Not the most beautiful code in the world. Do you have a better idea?

**Comment by VincentChalnot** [comment]
Merged manually

**Pull request closed without merging**